README.md

```diff
@@ -168,7 +168,7 @@ The duration (in milliseconds) of the scrolling animation.
 *Default:* `500` 
 
 #### easing 
-The easing to be used when animating. Read more in the [Easing section](#easing1). 
+The easing to be used when animating. Read more in the [Easing section](#easing-1). 
```

Comment:
This is a tricky one, because github uses`easing-1` but docute (the docs website) generates it as `easing1`. My idea is to just set the `## Easing` part to `<h2 id="easing-detailed">Easing</h2>` and use that as the id to be consistent on both github and the docs site. Can you please add these changes, so we can merge it all at once?